addons/web/static/src/views/utils.js

```diff
@@ -263,12 +263,12 @@ export function toStringExpression(str) {
 }
 
 /**
- * Generate a unique identifier.
+ * Generate a unique identifier (64 bits) in hexadecimal.
  *
  * @returns {string}
  */
 export function uuid() {
-    const array = new Uint8Array(16);
+    const array = new Uint8Array(8);
```

Comment:
"[IMP] web, base: shorten the properties name": would give an example in the commit message, so that people know what kind of name we have befoer / after :)